### ISSUE TITLE
Smart Fields not load on custom Related Data (with smart relationships)

### ISSUE THREAD

**Issue by GautierT** [ForestAdmin/forest-express-mongoose#82]
Hi,

I have created multiple smart fields for a model.
I then created a smart relationships to display this model on related data for an other model but here, i can see the columns for the smart fields but it's empty. The data are not loaded for this custom fields.

Thanks

**Comment by arnaudbesnier** [comment]
Hi @GautierT,

Please add a piece of code that represents you Smart Relationship getter implementation.
Without it we cannot help you.

**Comment by GautierT** [comment]
```
Liana.collection('letters', {
  actions: [{
    name: 'Send',
    fields: [
      {field: 'Production', type: 'String', description: ''}
    ]
  }],
  fields: [{
    field: 'Price Total',
    type: 'String',
    get: async function (object) {
      //debug('object : ', object);
      return get(object, 'price.total', 0)
    }
  }]
})
```

**Comment by GautierT** [comment]
Sometimes (one over 10) the smart fields are fetch and visible. 
I think it's a problem in Ember. But i have no error in the console.

**Comment by arnaudbesnier** [comment]
The Smart Field values injection is not made here (it is not the Serializer that does that job in the current liana).

To make the magic happen we should do improve this and it would make your code work perfectly.

By the way, is it a special relationship you have between `users` and `letters`? I don't get why Forest does not detect this relationship automatically.

**Comment by GautierT** [comment]
Yes, it's a special one. `Users` does'n't contain an array of `letters`.

Why is it working sometimes ?
Working : 
![image](https://user-images.githubusercontent.com/6089653/30106803-f30c299a-92fc-11e7-955b-6d50695422e9.png)

Not working :
![image](https://user-images.githubusercontent.com/6089653/30106821-043f3310-92fd-11e7-86d5-f01318ffeaf3.png)
